### PR TITLE
🐛 transcription fix for global_aggs

### PR DESCRIPTION
**Pull request by alex-wilmer**
1) add ES_LOG variable to change server terminal output (thanks @jgnieuwhof )
2) fix "aggregations_should_filter_themselves" to handle agg panel values

### CODE REVIEW

File: modules/middleware/test/global_aggs.test.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+};
+
+test('build_aggregations should handle nested aggregations', () => {
```
Comment (hlminh2000) [inline modules/middleware/test/global_aggs.test.js]: maybe a description like `"aggregations_filter_themselves: false", should give global aggs`

File: modules/middleware/src/aggregations.js [hunk 2]
```diff
@@ -604,17 +606,17 @@ export default class AggregationProcessor {
...
-          const innerValues = filteredEntries.map(item => item[1]);
-          v = {
-            [k]: innerValues,
-          };
-          break;
+          // original python code
+          /*
+          b = { k: innerValue for k, innerValue in v.items() if re.match(r"{}(:(stats|histogram))?".format(field), k) is not None}
+          */
+          v = Object.entries(v).reduce((acc, [k, v]) => {
```
Comment (hlminh2000) [inline modules/middleware/src/aggregations.js]: could we name the value in side the reduce something else? Was a little confusing as `v` is also used outside

Comment (alex-wilmer) [reply modules/middleware/src/aggregations.js]: confusing you say! yes this was horribly confusing and I kept it for some reason. will fix

File: modules/middleware/test/global_aggs.test.js (hunk 1 above)
Comment (alex-wilmer) [reply modules/middleware/test/global_aggs.test.js]: thanks nice catch

**Pull request merged**